lms/djangoapps/discussion_api/tests/test_api.py

```diff
@@ -313,13 +313,17 @@ def test_access_control(self):
         self.assertEqual(staff_actual, staff_expected)
 
 
-@httpretty.activate
 class GetThreadListTest(CommentsServiceMockMixin, ModuleStoreTestCase):
     """Test for get_thread_list"""
     def setUp(self):
         super(GetThreadListTest, self).setUp()
+        httpretty.enable()
+        self.addCleanup(httpretty.disable)
```

Comment:
FYI, I had to remove the httpretty.activate decorator and use this instead because otherwise the effect of register_user_response was reset immediately after setUp